samples/sample_encode/include/pipeline_encode.h

```diff
@@ -182,12 +185,88 @@ struct sInputParams
 
 };
 
+struct bufSet
```

Comment:
looks to be overcomplicated for this sample. In sample FEI work with ext buffers is important part, but here it shouldn't occupy so many code lines from my point

Reply:
yes, currently only one extension buffer to be used for sample encode. Here define the struct in case more extension buffer involved in the future.